addons/block_code/ui/main_panel.gd

```diff
@@ -117,7 +117,7 @@ func switch_block_code_node(block_code_node: BlockCode):
 	_delete_node_button.disabled = _current_block_code_node == null
 	if _current_block_code_node != null:
 		_try_migration()
-	_picker.block_script_selected(block_script)
+	_picker.block_script_selected(block_script, _current_block_code_node.get_parent() if _current_block_code_node else null)
```

Comment:
Nit: this would be a bit more readable without a ternary operator. Or at least moving the conditional to a variable:
```
var parent_node: Node = _current_block_code_node.get_parent() if _current_block_code_node else null
_picker.block_script_selected(block_script, parent_node)
```